tests/vmi_networking_test.go

```diff
@@ -828,26 +828,59 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				if !tests.HasLiveMigration() {
 					Skip("LiveMigration feature gate is not enabled in kubevirt-config")
 				}
+			})
+
+			AfterEach(func() {
+				if vmi != nil {
+					By("Delete VMI")
+					Expect(virtClient.VirtualMachineInstance(vmi.Namespace).Delete(vmi.Name, &v13.DeleteOptions{})).To(Succeed())
+
+					Eventually(func() error {
+						_, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &v13.GetOptions{})
+						return err
+					}, time.Minute, time.Second).Should(
+						SatisfyAll(HaveOccurred(), WithTransform(errors.IsNotFound, BeTrue())),
+						"The VMI should be gone within the given timeout",
+					)
+				}
+			})
+
+			table.DescribeTable("[Conformance] preserves connectivity", func(ipFamily k8sv1.IPFamily) {
```

Comment:
Between 30 to 50 percent of the code is conditioned by the ip-family. I do not think it is worth a table, a simple one would have worked fine. The duplication is not high and has almost no logic in it.

Reply:
7 lines are conditioned by ip family, 26 lines are common. I think your percentage calculation is not accurate.

Reply:
I count 15 in conditions and the condition used 3 times in the body.
If we count the overhead of the table itself, I really find this table usage unneeded.

I do not like it, but I am not going to block this PR over this.